back-end/handlers/hostHelpers.ts

```diff
@@ -150,10 +142,35 @@ const hostPreAnswer = async (gameId: number, io: Server): Promise<void> => {
   setTimeout(hostShowAnswer, PRE_ANSWER_MS, gameId, io);
 }
 
+const handleTiebreakerAnswers = async (allPlayers: IPlayer[], quizQIndex, correctQIndex): Promise<void> => {
```

Comment:
some ridiculous logic to handle all answers and ensure there is not another tie. basically, one of the tied players _has_ to get a bonus in one way or another.